plugin.xml

```diff
@@ -54,11 +54,17 @@
       <allow-navigation href="https://localhost" subdomains="true" />
       <allow-navigation href="https://*.force.com" subdomains="true" />
       <allow-navigation href="https://*.salesforce.com" subdomains="true" />
+      <allow-navigation href="https://*.visualforce.com" subdomains="true" />
+      <allow-navigation href="https://*.documentforce.com" subdomains="true" />
+      <allow-navigation href="https://*.salesforce-communities.com" subdomains="true" />
```

Comment:
Same question as the other repos -> what's `salesforce-communities.com`? I've never heard of it - I don't think we own that domain. That sounds like a custom community domain that a customer might have created.